code/modules/shuttle/ftl.dm

```diff
@@ -138,6 +178,8 @@
 			var/list/ports_list = list()
 			data["ports"] = ports_list
 			for(var/obj/docking_port/stationary/D in SSstarmap.current_planet.docks)
+				if(D.planet_dock || D.name == "FOB Landing Zone") //Only the FOB can land on planets now
+					continue
```

Comment:
just remove planet landing functionality instead of this which will leave the button in the ui

Reply:
Old code from the two planet docks era. Woops.